crt_portal/cts_forms/tests/tests.py

```diff
@@ -88,8 +74,8 @@ def test_Location_invalid(self):
         )
 
     def test_Class_valid(self):
-        form = ProtectedClassForm(data={
-            'protected_class': ProtectedClass.objects.filter(protected_class__in=PROTECTED_CLASS_CHOICES),
+        form = ProtectedClassForm({
+            'protected_class': ProtectedClass.objects.all(),
```

Comment:
I don't think we will want all objects especially over time as this list changes and we need to preserve the previous data